### DISCUSSION TITLE
size property on datepicker exists or not?

### DISCUSSION THREAD

**Discussion by denilson157** [rsuite/rsuite#1920] [Q&A] [answered]
Hi, im using the 5.0.0 beta version's of rsuite. 
Im refactoring my code in typescript and when i started to refactoring my fields modules, the intellisense of visual studio shows me an error saying the size property is not assignable. But in javascript works great!

In [Datepicker documentation](https://rsuitejs.com/components/date-picker/#Props) doesn't exists size property but in a javascript file, if i send property size it works.

I don't know if this a bug or i some what i'm doing wrong

**Comment by simonguo** [comment, marked as answer]
Thank you for finding the problem, it has been fixed.  https://github.com/rsuite/rsuite/pull/1926
